modules/ppcp-agentic-commerce/src/Endpoint/AgenticRestEndpoint.php

```diff
@@ -100,7 +100,7 @@ public function check_permission( WP_REST_Request $request ) {
 	 * @param int          $status_code HTTP status code.
 	 * @return WP_REST_Response The successful response.
 	 */
-	protected function cart_details( CartResponse $cart, int $status_code = 200 ): WP_REST_Response {
+	protected function cart_details( CartResponse $cart, int $status_code ): WP_REST_Response {
 		$this->logger->info( "[REST] $status_code Response", $cart->to_array() );
```

Comment:
Im aware this comes from previous code and is not refactored. But are we sure we want to log all the requests?

Reply:
Yes, this module is in active development and we have no other means to understand what happens.

I'd even argue to keep it in production (at least for a while) as agentic integration is a communication between two servers where we have no other means to debug or investigate behavior using a browser, like we have for most other features